### DISCUSSION TITLE
Touch without consume

### DISCUSSION THREAD

**Discussion by cVecchioTheOld** [skydoves/Balloon#304] [Q&A]
Hi @skydoves,

I'm trying to create a tooltip that is dismissed without consuming the user click but I'm not able to do it.
For instance when I show a tooltip on a button and I perform a click on another button I would like to both dismiss the tooltip and pass the click to clicked button.
Is there a way to do it?

Cheers,
Claudio

**Comment by skydoves** [comment]
Hi @cVecchioTheOld,
Could you expand on this? You can dismiss the tooltips with the `dismiss()` method.

**Reply by cVecchioTheOld** [reply]
For instance this is the tooltip that I created. 

![image](https://user-images.githubusercontent.com/48517311/150388621-2727f3b4-4662-45ea-8632-c36e5a9c8342.png)

when it is displayed if I click on a button of the action bar the tooltip is dismissed but I have to click again on button to perform the button related action.

What I would like to achieve is that when I click on a button then the tooltip is dismissed and button action is performed automatically

**Reply by SPapadatos** [reply]
I was looking to do the same thing, I know this is an old question but is there a solution to this?

To clarify: I would like to be able to show a tooltip to indicate to the user to click a button, but, have them be able to click the button on the first try. Not dismiss the tooltip then be able to click the button.

Thank you

**Reply by Kuglll** [reply]
@skydoves Any news here? I am facing the same problem. I tried with `.setShouldPassTouchEventToAnchor(true)`, but it does not seem to solve the issue.

**Reply by Kuglll** [reply]
Nvm, I found the solution. @cVecchioTheOld @SPapadatos I know this is very old thread, but here is the solution, if you are still looking for one: https://github.com/skydoves/Balloon/issues/288
